test/modules/layers/test_attention.py

```diff
@@ -125,22 +129,56 @@ def test_combine_multihead(self):
 
     def test_multi_head_attention(self):
         # New tensors because need unflattened shape
-        q = torch.randn(1, *self.input_shape, self.hidden_dim)
-        k = torch.randn(1, *self.input_shape, self.hidden_dim)
-        v = torch.randn(1, *self.input_shape, self.hidden_dim)
+        q = 2 * torch.ones(1, *self.input_shape, self.hidden_dim)
```

Comment:
is this change related?

Reply:
not directly, but since I changed the setUp that affected all the random tensors since they generate different values based on the order they are called... I figured it was too unstable so I made it a fixed input instead.